---

title: Our new resource update paper is out in Nucleic Acids Research!

author: GWAS Catalog team

date: Nov 20, 2024

description: We are very happy to announce the GWAS Catalog’s latest resource update paper in Nucleic Acids Research! We discuss standards for reusability, sustainability and diversity, new software features, and more.... 

slug: nar-resource-paper-2024

img: ...

---
**We are very happy to announce the GWAS Catalog’s [latest resource update paper](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae1070/7893318) in Nucleic Acids Research!**

Here’s a hint of what you can find inside:
The GWAS-SSF standard for full genome-wide summary statistics data and metadata has been fully implemented in our submission pipeline. More than 30k files have been submitted containing ALL mandatory data & metadata fields, on top of 50k files in the previous format.  We’ve also scaled our harmonisation pipeline to keep up with the increased rate of data submission - now 92% of all summary statistics with the necessary data content have a [harmonised version](https://www.ebi.ac.uk/gwas/docs/methods/summary-statistics#_directories)  available. 

We have seen a huge growth in molecular quantitative trait GWAS (proteomic, lipidomic & metabolomic measurements) and are working with our colleagues at the Experimental Factor Ontology to improve mapping to chemical entities. 
<article-image src="nar-resource-paper-2024/Fig1.tif" alt="molecular-gwas"></article-image>

New features enable faster page loading and allow you to easily find GxE studies in our website and download files, preparing the way for anticipated growth in GxE data is anticipated to grow as the scale of exposome data expands. Thanks to funding from the [Human Ecosystems Transversal Theme](https://www.embl.org/about/info/human-ecosystems/) at EMBL-EBI!
<article-image src="nar-resource-paper-2024/Fig2.tif" alt="gwas-catalog-new-software"></article-image>


The lack of diversity in genomics data continues to be an issue! We share our thoughts on how to enable monitoring of diversity and inclusion in GWAS studies while taking care that the language we use to describe populations does not lead to misuse. [New documentation and terminology](https://www.ebi.ac.uk/gwas/population-descriptors) (labels instead of categories) guide our users towards correct interpretation.
Our analysis found that only around 50% of GWAS samples reported in the literature are being defined by genetic similarity measures, of those less than half clearly described a reference panel.  
<article-image src="nar-resource-paper-2024/SupFig3.png" alt="gwas-catalog"></article-image>

UKB is still the most used cohort in GWAS, accounting for around 30% of all studies in the Catalog! Unfortunately 70% of these restrict analysis to samples with the European population label, losing a significant amount of diversity. New inclusion of cohort names in Catalog data now enables this analysis, and in future linking out to rich cohort metadata. 
<article-image src="nar-resource-paper-2024/Fig3.tif" alt="gwas-catalog-diversity"></article-image>


Finally, we are now recognised as a Global Biodata Core Resource and Elixir Core Data resource!
<article-image src="nar-resource-paper-2024/core-resources.png" alt="gwas-catalog-diversity"></article-image>


Read the full article, [Cerezo et al 2024](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae1070/7893318), open access at NAR.


Comments, feedback…? Contact us via gwas-info@ebi.ac.uk
