__NUXT_JSONP__("/everything-you-need-to-know-about-the-new-gwas-catalog", (function(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z,A,B,C,D){return {data:[{article:{slug:"everything-you-need-to-know-about-the-new-gwas-catalog",description:r,title:"Everything you need to know about the new GWAS Catalog",author:"Annalisa Buniello",img:"blog-1.jpg",date:"2018-03-13T00:00:00.000Z",displayDate:"Jun 13, 2018",toc:[{id:s,depth:f,text:t},{id:u,depth:f,text:v},{id:w,depth:f,text:x},{id:y,depth:f,text:z}],body:{type:"root",children:[{type:b,tag:d,props:{},children:[{type:a,value:r}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The NHGRI-EBI GWAS Catalog, an awesome public resource of human genotype-phenotype associations from published GWAS analyses, is one of the data sources for manually curated genetic evidence for the Open Targets Platform. The GWAS Catalog data is widely used to prioritise new candidate loci, predict disease risk and understand disease mechanisms, with a number of users and applications that is always increasing."}]},{type:a,value:c},{type:b,tag:g,props:{src:"everything-you-need-to-know-about-the-new-gwas-catalog\u002FFINAL_USERS_withOT-2.png",alt:h},children:[{type:a,value:c}]},{type:a,value:c},{type:b,tag:i,props:{id:s},children:[{type:b,tag:e,props:{href:"#new-and-expanded-gwas-catalog-to-improve-target-identification",ariaHidden:j,tabIndex:k},children:[{type:b,tag:l,props:{className:[m,n]},children:[]}]},{type:a,value:t}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"I am happy to announce an expansion of the GWAS Catalog scientific scope to host non genome-wide studies. During its first 10 years of life, in fact, the GWAS Catalog has only included array studies with genome-wide coverage, therefore curation of exome and targeted array studies were outside its scientific scope. These data are however valuable to Open Targetsâ€™ scientific goals of target identification and prioritisation, so much so they decided to invest in a new project aiming to expand the scope of the Catalog to include targeted arrays, such as MetaboChip, ImmunoChip, and Exome array. The new GWAS will now allow variants associated with immunologic, metabolic and oncologic phenotypes to be included in the Open Targets Platform for the expansion of target identification and selection in these traits."}]},{type:a,value:c},{type:b,tag:i,props:{id:u},children:[{type:b,tag:e,props:{href:"#selecting-and-curating-non-genome-wide-array-studies",ariaHidden:j,tabIndex:k},children:[{type:b,tag:l,props:{className:[m,n]},children:[]}]},{type:a,value:v}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"As a first step, Open Targets selected almost 60 publications for inclusion in the GWAS Catalog. I then extracted a total of 120 new independent association studies and 823 SNP-trait associations from these publications. These are impressive numbers, and give you an idea of how many studies and SNP-trait associations (and potential targets) we were missing by not including targeted array studies in the GWAS Catalog!"}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The final step for me to do was to map all these traits, diseases and phenotypes to EFO terms, the ontology used by\n"},{type:b,tag:e,props:{href:"https:\u002F\u002Fdocs.targetvalidation.org\u002Ffaq\u002Fdescribing-diseases",rel:[A,B,C],target:D},children:[{type:a,value:"Open Targets to describe its diseases"}]}]},{type:a,value:c},{type:b,tag:i,props:{id:w},children:[{type:b,tag:e,props:{href:"#modifying-the-gwas-catalog-user-interface",ariaHidden:j,tabIndex:k},children:[{type:b,tag:l,props:{className:[m,n]},children:[]}]},{type:a,value:x}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"The GWAS Catalog team have changed the way the user interface works to support searching, displaying, filtering and download of targeted and exome array studies. "},{type:b,tag:e,props:{href:"https:\u002F\u002Fwww.ebi.ac.uk\u002Fgwas\u002Fsearch?query=celiac%20disease",rel:[A,B,C],target:D},children:[{type:a,value:"Try it out using \"celiac disease\""}]},{type:a,value:", for example. The interface will now give you:"}]},{type:a,value:c},{type:b,tag:o,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:b,tag:q,props:{},children:[{type:a,value:"Drop-down filter by genotyping technology to help you focus on your favourite array"}]}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:g,props:{src:"everything-you-need-to-know-about-the-new-gwas-catalog\u002Ffilter1-5.png",alt:h},children:[{type:a,value:c}]},{type:a,value:c},{type:b,tag:o,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:b,tag:q,props:{},children:[{type:a,value:"Small \"target\" icon (designed for us by the EMBL-EBI media team. Thanks for that!) in the study accession number column for all targeted array studies prioritised by Open Targets"}]}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:g,props:{src:"everything-you-need-to-know-about-the-new-gwas-catalog\u002Ftarget-1.png",alt:h},children:[{type:a,value:c}]},{type:a,value:c},{type:b,tag:o,props:{},children:[{type:a,value:c},{type:b,tag:p,props:{},children:[{type:b,tag:q,props:{},children:[{type:a,value:"New \"genotyping technology\" shown for study, e.g. genome-wide genotyping array, exome-wide genotyping array or targeted genotyping array with an optional field for additional array information, for example ImmunoChip or MetaboChip"}]}]},{type:a,value:c}]},{type:a,value:c},{type:b,tag:g,props:{src:"everything-you-need-to-know-about-the-new-gwas-catalog\u002Fgen-tech-1.png",alt:h},children:[{type:a,value:c}]},{type:a,value:c},{type:b,tag:i,props:{id:y},children:[{type:b,tag:e,props:{href:"#stay-tuned",ariaHidden:j,tabIndex:k},children:[{type:b,tag:l,props:{className:[m,n]},children:[]}]},{type:a,value:z}]},{type:a,value:c},{type:b,tag:d,props:{},children:[{type:a,value:"This is an important milestone for Open Targets and the GWAS Catalog, and a lot more is yet to come. The next step for the collaboration between GWAS and Open Targets will be to develop a comprehensive database of all available GWAS full summary statistics stored in a common format and harmonised across studies to enable searching, easy comparison and downstream analysis. This effort will generate more comprehensive new raw genetic data accessible by the entire research community through the GWAS Catalog, the Open Targets Platform and additional resources the Open Targets team are working on."}]}]},dir:"\u002Fblog",path:"\u002Fblog\u002Feverything-you-need-to-know-about-the-new-gwas-catalog",extension:".md",createdAt:"2025-11-10T13:12:41.745Z",updatedAt:"2025-11-10T13:39:23.852Z"}}],fetch:{},mutations:void 0}}("text","element","\n","p","a",3,"article-image","gwas-catalog","h3","true",-1,"span","icon","icon-link","ul","li","strong","Genetic evidence could double the rate of success in drug targets identification - so the scientific community predicts. With this in mind, it is crucial for Open Targets to be able to feed as many SNP-trait associations as possible into its pipeline for systematic target identification and validation.","new-and-expanded-gwas-catalog-to-improve-target-identification","New and expanded GWAS Catalog to improve target identification","selecting-and-curating-non-genome-wide-array-studies","Selecting and curating non-genome-wide array studies","modifying-the-gwas-catalog-user-interface","Modifying the GWAS Catalog user interface","stay-tuned","Stay tuned!","nofollow","noopener","noreferrer","_blank")));